modules/primo/src/Primo/BriefSearch/Document.php

```diff
@@ -280,4 +280,23 @@ public function getOnlineUrl() {
     return $this->nodeValue($nodes);
   }
 
+  /**
+   * Get a local search field (lsr) by number.
+   *
+   * @param $number
+   *   The search field number to get.
+   *
+   * @return string
+   *   The local search field value.
+   */
+  public function getLocalSearchField($number) {
+    $nodes = $this->recordXpath($this->recordId, '//primo:search/primo:lsr'
+    . $number);
+    $node_values = array_map(function($node) {
+      return $node;
+    }, $this->nodeValues($nodes));
+
+    return implode(', ', $node_values);
```

Comment:
I do not think merging at this level is the right approach. It assumes something about how the caller will use the result. 

I would prefer that we return an array of strings. Then the `ObjectMapper` can decide to implode the values into a single string for display purposes.